guides/app-guides/README.md

```diff
@@ -76,10 +41,8 @@ Here are the steps to impersonate another address:
 1. Connect your wallet.
 2. Click on `Settings` in the top left corner.
 3. At the bottom of the Settings section, there is an eye icon with an input to the right. Enter any address, and the connection will change.
-   ![Impersonator](../../.gitbook/assets/impersonator.png)
-   ![Impersonator connected](../../.gitbook/assets/impersonator-connected-2.png)
+   ![Impersonator](/.gitbook/assets/impersonator.png)
```

Comment:
Let's make sure we're going to the line for this otherwise will be weirdly displayed